google-datacatalog-hive-connector/tests/google/datacatalog_connectors/hive/prepare/assembled_entry_factory_test.py

```diff
@@ -72,6 +72,60 @@ def test_database_metadata_should_be_converted_to_assembled_entries(
         self.assertEqual(expected_created_entries_len, created_entries_len)
         self.__assert_created_entry_fields(assembled_entries)
 
+    def test_database_metadata_invalid_ids_should_be_converted_to_assembled_entries(  # noqa:E501
+            self, entry_path):  # noqa
+
+        entry_path.return_value = \
+            AssembledEntryFactoryTestCase.__MOCKED_ENTRY_PATH
```

Comment:
```suggestion
            self.__MOCKED_ENTRY_PATH
```
WDYT about using `self` to turn this and similar statements less verbose?